x-pack/plugins/security_solution/cypress/integration/detection_rules/indicator_match_rule.spec.ts

```diff
@@ -92,104 +110,337 @@ describe('Detection rules, Indicator Match', () => {
-      getDetails(RISK_SCORE_DETAILS).should('have.text', newThreatIndicatorRule.riskScore);
-      getDetails(REFERENCE_URLS_DETAILS).should((details) => {
-        expect(removeExternalLinkText(details.text())).equal(expectedUrls);
-      });
-      getDetails(FALSE_POSITIVES_DETAILS).should('have.text', expectedFalsePositives);
-      getDetails(MITRE_ATTACK_DETAILS).should((mitre) => {
-        expect(removeExternalLinkText(mitre.text())).equal(expectedMitre);
-      });
-      getDetails(TAGS_DETAILS).should('have.text', expectedTags);
+
+    describe('custom query input', () => {
+      it('Has a default set of *:*', () => {
+        getCustomQueryInput().should('have.text', '*:*');
+      });
+
+      it('Shows invalidation text if text is removed', () => {
+        getCustomQueryInput().type('{selectall}{del}');
+        getCustomQueryInvalidationText().should('exist');
+      });
     });
-    cy.get(INVESTIGATION_NOTES_TOGGLE).click({ force: true });
-    cy.get(ABOUT_INVESTIGATION_NOTES).should('have.text', INVESTIGATION_NOTES_MARKDOWN);
-
-    cy.get(DEFINITION_DETAILS).within(() => {
-      getDetails(INDEX_PATTERNS_DETAILS).should(
-        'have.text',
-        newThreatIndicatorRule.index!.join('')
-      );
-      getDetails(CUSTOM_QUERY_DETAILS).should('have.text', '*:*');
-      getDetails(RULE_TYPE_DETAILS).should('have.text', 'Indicator Match');
-      getDetails(TIMELINE_TEMPLATE_DETAILS).should('have.text', 'None');
-      getDetails(INDICATOR_INDEX_PATTERNS).should(
-        'have.text',
-        newThreatIndicatorRule.indicatorIndexPattern.join('')
-      );
-      getDetails(INDICATOR_MAPPING).should(
-        'have.text',
-        `${newThreatIndicatorRule.indicatorMapping} MATCHES ${newThreatIndicatorRule.indicatorIndexField}`
-      );
-      getDetails(INDICATOR_INDEX_QUERY).should('have.text', '*:*');
+
+    describe('custom indicator query input', () => {
+      it('Has a default set of *:*', () => {
+        getCustomIndicatorQueryInput().should('have.text', '*:*');
+      });
+
+      it('Shows invalidation text if text is removed', () => {
+        getCustomIndicatorQueryInput().type('{selectall}{del}');
+        getCustomQueryInvalidationText().should('exist');
+      });
     });
 
-    cy.get(SCHEDULE_DETAILS).within(() => {
-      getDetails(RUNS_EVERY_DETAILS).should(
-        'have.text',
-        `${newThreatIndicatorRule.runsEvery.interval}${newThreatIndicatorRule.runsEvery.type}`
-      );
-      getDetails(ADDITIONAL_LOOK_BACK_DETAILS).should(
-        'have.text',
-        `${newThreatIndicatorRule.lookBack.interval}${newThreatIndicatorRule.lookBack.type}`
-      );
+    describe('Indicator mapping', () => {
+      beforeEach(() => {
+        fillIndexAndIndicatorIndexPattern(
+          newThreatIndicatorRule.index,
+          newThreatIndicatorRule.indicatorIndexPattern
+        );
+      });
+
+      it('Does NOT show invalidation text on initial page load', () => {
```

Comment:
Broad comment: these narrowly-focused tests are great for legibility and test isolation but likely terrible for CI speed. My preference for these types of tests is to walk through a particular user workflow, making assertions along the way until we get to a terminal point, e.g.:

* sign in, define rule, about rule, _go back to define step and assert values are repopulated_, create rule, assert rule created

This seems to be the predominant pattern in our cypress suite right now, so these new tests are a divergence from that. Neither's objectively correct, certainly, and I don't think these _need_ to change, but I wanted to call out the difference. @MadameSheema may have a stronger opinion here 😉